src/components/InfoViewer/InfoViewer.scss

```diff
@@ -27,11 +27,10 @@
 
     &__label {
         display: flex;
-        flex: 1 1 auto;
+        flex: 0 1 auto;
         align-items: baseline;
 
         min-width: 200px;
-        max-width: 200px;
```

Comment:
to allow longer labels widen the block, like this
![](https://jing.yandex-team.ru/files/trshkv/2022-08-17_14-52-06.png =364,)